pycheribuild/projects/cross/qt5.py

```diff
@@ -753,6 +755,10 @@ class BuildICU4C(CrossCompileAutotoolsProject):
     native_install_dir = DefaultInstallDir.CHERI_SDK
     make_kind = MakeCommandKind.GnuMake
     needs_native_build_for_crosscompile = True
+    supported_architectures = list(
```

Comment:
I think we'd normally spell this `supported_architectures = CompilationTargets.ALL_CHERIBSD_TARGETS_WITH_HYBRID + [CompilationTargets.NATIVE]`